src/TypeScript/EslintBridgeClient/EslintBridgeProcess.cs

```diff
@@ -152,7 +158,8 @@ private void StartServer()
 
         private void Process_Exited(object sender, EventArgs e)
         {
-            Process = null;
+            logger.LogDebug($"[eslint-bridge] [process id: {processId}] Process exited.", processId);
+            ClearProcessData();
```

Comment:
I wonder if we have (and always had) a race condition here.
It's not protected in a lock, so it depends on when this event is called.
i.e.
1) Thread A: Process A has crashed, this event is raised.
2) Thread B: Calls EslintBridgeProcess.Start, gets to `Process.HasExited=true`
3) Thread B: Starts a new process and calls `Process = new Process { StartInfo = psi }`
4) Thread A: sets `Process = null`

Reply:
Yep, looks possible. I'm not sure what the consequences of that series of events would be :
* the external node process would still be running, so the the node process would still be up and would be able to reply to ThreadB.
* the _next_ request to analyze would then start another instance of the node server, since Process = null.
So we'd have an orphaned node process that is still running?

Reply:
Yep. I think that's what would happen. It could explain why I saw two "started" logs sometimes, even though I didn't open a JS file, only TS.